src/option_parser.c

```diff
@@ -109,6 +110,19 @@ char *get_value(char *section, char *key)
         return NULL;
 }
 
+char *ini_get_path(char *section, char *key, const char *def)
+{
+        char *path = ini_get_string(section, key, def);
+
+        wordexp_t replaced_path;
+        if (path && 0 == wordexp(path, &replaced_path, WRDE_NOCMD)){
```

Comment:
Some more error handling here would be nice, especially for `WRDE_BADVAL` and `WRDE_CMDSUM`.